flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -273,22 +278,22 @@ private void testKeyedListStateUpgrade(
 			// make sure that reading and writing each key state works with the new serializer
 			backend.setCurrentKey(1);
 			Iterator<TestType> iterable1 = listState.get().iterator();
-			Assert.assertEquals(new TestType("key-1", 1), iterable1.next());
-			Assert.assertEquals(new TestType("key-1", 2), iterable1.next());
-			Assert.assertEquals(new TestType("key-1", 3), iterable1.next());
+			assertEquals(new TestType("key-1", 1), iterable1.next());
+			assertEquals(new TestType("key-1", 2), iterable1.next());
+			assertEquals(new TestType("key-1", 3), iterable1.next());
 			Assert.assertFalse(iterable1.hasNext());
 			listState.add(new TestType("new-key-1", 123));
 
 			backend.setCurrentKey(2);
 			Iterator<TestType> iterable2 = listState.get().iterator();
-			Assert.assertEquals(new TestType("key-2", 1), iterable2.next());
+			assertEquals(new TestType("key-2", 1), iterable2.next());
 			Assert.assertFalse(iterable2.hasNext());
 			listState.add(new TestType("new-key-2", 456));
 
 			backend.setCurrentKey(3);
 			Iterator<TestType> iterable3 = listState.get().iterator();
-			Assert.assertEquals(new TestType("key-3", 1), iterable3.next());
-			Assert.assertEquals(new TestType("key-3", 2), iterable3.next());
+			assertEquals(new TestType("key-3", 1), iterable3.next());
```

Comment:
in the future, please try to avoid these changes, which are irrelevant to the fix.
Or at least include them as a separate commit.